openage/__main__.py

```diff
@@ -103,10 +102,17 @@ def main(argv=None):
     if args.no_devmode and args.devmode:
         cli.error("can't force enable and disable devmode at the same time")
 
-    if args.no_devmode:
-        config.DEVMODE = False
-    if args.devmode:
-        config.DEVMODE = True
+    try:
+        from . import config
+
+        if args.no_devmode:
+            config.DEVMODE = False
+        if args.devmode:
+            config.DEVMODE = True
+    except ImportError:
+        if args.no_devmode or args.devmode:
+            print("code was not yet generated, "
```

Comment:
I didn't knew this syntax existed for string concatenation existed in python, still stuck in 2.7.